### PR TITLE
WisdomSoftware.screenhunter version 7.0.1427

### PR DESCRIPTION
**Pull request by solomoncyj**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/64509)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/w/WisdomSoftware/screenhunter/7.0.1427/WisdomSoftware.screenhunter.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: WisdomSoftware.screenhunter
```
Comment (ItzLevvie) [inline manifests/w/WisdomSoftware/screenhunter/7.0.1427/WisdomSoftware.screenhunter.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: WisdomSoftware.ScreenHunter
```

File: manifests/w/WisdomSoftware/screenhunter/7.0.1427/WisdomSoftware.screenhunter.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: WisdomSoftware.screenhunter
```
Comment (ItzLevvie) [inline manifests/w/WisdomSoftware/screenhunter/7.0.1427/WisdomSoftware.screenhunter.installer.yaml:4]:
```suggestion
PackageIdentifier: WisdomSoftware.ScreenHunter
```

File: manifests/w/WisdomSoftware/screenhunter/7.0.1427/WisdomSoftware.screenhunter.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: WisdomSoftware.screenhunter
```
Comment (ItzLevvie) [inline manifests/w/WisdomSoftware/screenhunter/7.0.1427/WisdomSoftware.screenhunter.yaml:4]:
```suggestion
PackageIdentifier: WisdomSoftware.ScreenHunter
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by solomoncyj** [comment]
vs code is acting up

**Pull request closed without merging**